modules/test/playwright/tests/export-import-web/pages/ExportImportPage.ts

```diff
@@ -21,6 +21,8 @@ export class ExportImportPage {
 	readonly newImportProcess: Locator;
 	readonly page: Page;
 	readonly productMenuPage: ProductMenuPage;
+	readonly rangeDateRangeRadioButton: Locator;
```

Comment:
Also, I know that we are using `ExportImportPage` inside `CompanyExportImportPage` in other cases. (See `CompanyExportImportPage` in 9a92952ac13cb431e4c486cc61dee75c5c04f1f9), but I don't know if its the right thing...
Maybe @dannielraposo knows if this is appropiate or we should create it inside of `companyExportImportPage`